src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchBar">
+      <div class="container">
+
+        <form 
+          action="/"
+          method="POST"
+          data-qa="big"
+        >
+          <div class="formInput">
+
+            <span class="bIcon">
```

Comment:
```suggestion
          <div class="formInput">
            <span class="bIcon">
```

![image](https://user-images.githubusercontent.com/80700247/220905037-d5bafd29-adee-4de2-a913-532ca676b58b.png)

Fix everywhere